README.md

```diff
@@ -12,75 +10,41 @@
+* `_query` (object) - Contains the query parameters.
+* `_headers` (object) - Contains the headers of the received request.
+* `_method` (string, `POST` or `GET`) - Indicates the HTTP method of the received request.
+* `_url` (string) - The full URL that was received.
+
+#### Webhook Response
+By default, the webhook URL will respond with the following structure: 
+```json
 {
-    "recievedBody": "recievedBody",
-    "_query": {},
-    "_headers": {
-      "content-type": "application/json",
-      "accept": "*/*",
-      "accept-encoding": "gzip, deflate"
-    },
-    "_method": "POST",
-    "_url": "/hook/5d691738cb5a286adc1e68e2"
-  }
+    "requestId": "86ad47dfbce4a8ae8a1eb505b85d8bd5",
+    "message": "thank you"
+}
 ```
 
-## Actions 
-  ### Send data
-  Simply sends data it receives as an input to a URL provided.
-  
-  WebHook action can also be used to troubleshoot many processes to see the outcome. 
-  For example one could create Invoices (in Salesforce) to Webhook flow and configure the Webhook with a url created in https://webhook.site or with any similar services.
-
-  #### List of Expected Config fields
-  * **[required]** **HTTP Verb**
-    * **POST**. The WebHook component can POST information to preconfigured WebHook address. This action could be used for different purposes. For example WebHook can be used to inform your custom connector about an event which it waits to work.
-    * **PUT**. The WebHook component can also PUT a specific preconfigured JSON into specific address where the process will not be handled by the server. For this reason the "Output JSON Sample" field can be used.
-  * **[required]** **URI**. This is the address to send WebHook.
-  * **[not required]** **Secret**. This is an optional field to authenticate WebHook POST. There maybe cases when a special password or a secret might be required. For example the WebHook address was generated explicitly with a password so that to prevent any third parties to use it. This could be your specific WebHook address that you use to send your Wordpress posts into your server.
-  
-  ![Send Data config fields](https://user-images.githubusercontent.com/8449044/61964168-eff1b200-afd5-11e9-8928-2890c3360d13.png)
-
-  #### Expected output metadata
-   [Output schema](lib/schemas/base64.out.json)
-     
-   Example:
-   ```metadata json
-  {
-      "recievedBody": "recievedBody",
-      "_query": {},
-      "_headers": {
-        "content-type": "application/json",
-        "accept": "*/*",
-        "accept-encoding": "gzip, deflate"
-      },
-      "_method": "POST",
-      "_url": "/hook/5d691738cb5a286adc1e68e2"
-    }
-  ```
-
-## Known limitations
+However, you can specify the exact content of the reply using the `HTTP Reply` component. In this case, you will receive a reply only when the message reaches the step where the `HTTP Reply` is used, or if an error occurs in other steps between the `Webhook` step and the `HTTP Reply`.
```

Comment:
Would be nice to put the link to the HTTP Reply component (on the docs site)

Reply:
done